Libraries/Core/Timers/JSTimers.js

```diff
@@ -149,6 +178,12 @@ const JSTimers = {
       JSTimersExecution.requestIdleCallbacks.splice(index, 1);
     }
 
+    const timeout = JSTimersExecution.requestIdleCallbackTimeouts[timerID];
```

Comment:
Can you call this `timeoutId` to make it clear it's not the same as the other timeout var we use that is the time of the timeout.